src/vma/dev/qp_mgr.h

```diff
@@ -55,6 +55,9 @@
 REVIEW: verify can remove: #include "vma/dev/ring.h"
 #endif
 
+//#define ALIGN_WR_UP(_num_wr_) 		(max(32, ((_num_wr_ + 0xf) & ~(0xf))))
+#define ALIGN_WR_DOWN(_num_wr_) 		(max(32, ((_num_wr_      ) & ~(0xf))))
```

Comment:
This is defined both here and and in _src/vma/dev/ring_simple.cpp_. Please put it in a single common H file.

#### Introduced in:
MP_RQ: add cq_mgr_mp and qp_mgr_mp